src/ray/gcs/gcs_server/gcs_kv_manager.h

```diff
@@ -39,6 +39,16 @@ class InternalKVInterface {
                    const std::string &key,
                    std::function<void(std::optional<std::string>)> callback) = 0;
 
+  /// Get the values associated with `key/namespace` pairs.
+  ///
+  /// \param ns The namespace of the key.
+  /// \param key The key to fetch.
+  /// \param callback Callback function.
+  virtual void MultiGet(
```

Comment:
Can we/should we provide a default implementation of multiget here that just calls `Get` in a loop?